### DISCUSSION TITLE
Database Queries | MySQL CLI Script Files

### DISCUSSION THREAD

**Discussion by nashalysf** [BraianPerez97/CoquiArts#26] [General]
# Running Queries: DB, Schema and Seed Files:

Enter mysql cli : `mysql -u root `

* Once in cl run `source` followed by the file path `db/db_setup.sql` to structure the database
`mysql> source db/db_setup.sql;`

* Then run again the command but to the `schema.sql` file to structure the tables
`mysql> source db/schema.sql;`

* Finally, run again the command but to the `seeds.sql` file to seed data
`mysql> source db/seeds.sql;`

Equivalent to:
`DROP DATABASE IF EXISTS coquiArts;`
`CREATE DATABASE IF NOT EXISTS coquiArts;`
`USE coquiArts;`
`CREATE TABLE users (
id INTEGER AUTO_INCREMENT PRIMARY KEY,
    first_name VARCHAR(30) NOT NULL,
    last_name VARCHAR(30),
    email VARCHAR(31) NOT NULL,
    passwd VARCHAR(60) NOT NULL
);` and `CREATE TABLE categories (
    id INTEGER AUTO_INCREMENT PRIMARY KEY, 
    name VARCHAR(50) NOT NULL, 
    description TEXT
    );`
`INSERT INTO user (first_name, last_name, email, passwd)
VALUES (...)`
`INSERT INTO categories (name, description)
VALUES(...)`

Now, check that the files run successfully through database: 
`USE coquiArts;`
`SHOW TABLES;`
`DESCRIBE user;`
`DESCRIBE categories;`

Once the creation of the db is complete, exit the program `quit`
`mysql> quit;`

Note: remember that in MySQL CLI all commands must end with a semi-colon `;`

**Comment by BraianPerez97** [comment]
Changed 'users' table to 'user'
